.circleci/config.yml

```diff
@@ -18,9 +18,7 @@ jobs:
           name: Install Python dependencies
           command: |
             pip --version
-            pip install --progress-bar off . \
-              -r ./dev/large-requirements.txt \
```

Comment:
We don't need `large-requirements.txt` anymore because we currently **lazily** import ML frameworks within functions.